src/BlazorStrap/Components/Carousel/BSCarousel.razor.cs

```diff
@@ -53,6 +53,10 @@ protected void onmouseout()
             if (PauseOnHover && _timer != null) { _timer.Start(); }
         }
 
+        internal void FireEvent(int newIndex, int OldIndex)
```

Comment:
Did you mean to have this empty?

Reply:
Events on Carousel are going to be pushed next release or later. Will require some rewrite to make them work properly.